core/src/main/scala/org/broadinstitute/dsde/rawls/Boot.scala

```diff
@@ -152,6 +152,7 @@ object Boot extends App with LazyLogging {
         shardedExecutionServiceCluster,
         conf.getInt("executionservice.batchSize"),
         gcsDAO.getBucketServiceAccountCredential,
+        util.toScalaDuration(conf.getDuration("executionservice.processInterval")),
```

Comment:
process interval: interval between processing batches of workflows
poll interval: interval between polling when you've not found any and need to check back later